common/shlibs

```diff
@@ -2674,7 +2674,7 @@ libsmpeg-0.4.so.0 smpeg-0.4.5_3
 libsmpeg2-2.0.so.0 smpeg2-2.0.0_1
 libedac.so.1 libedac-0.18_1
 libxlsreader.so.8 libxls-1.6.1_1
-libR.so R-4.0.2_1
+libR.so R-4.0.3_1
```

Comment:
Is there a breaking ABI change in this version of `libR.so`? If so, all of the `xrevshlib R` dependants need to be revbumped. If not, don't update `common/shlibs` with the new package version.

Reply:
updating shlibs without revbumping is actually fine but it depends on circumstances

usually you'd bump the version in shlibs if the new version introduced some API that packages are going to use (as in that case, the packages built against the newer version will require it, while packages built against the older one will keep working just fine)

Reply:
Sure, will change it. @q66 you're right there aren't any new APIs or major changes, was just unsure about whether to update shlibs when updating a package that provides them.